web-app/vue.config.js

```diff
@@ -32,5 +32,83 @@ module.exports = {
   },
   transpileDependencies: [
     'vuetify'
-  ]
+  ],
+  pwa: {
+    themeColor: "#2196F3",
```

Comment:
![Screenshot_20210721-195924](https://user-images.githubusercontent.com/2344456/126574895-d9891b12-457b-4f56-8bd1-d24dba920b04.jpg)

What about using the primary blue (#1976D2) from the default theme so it all blends together seamlessly?

Another thing we'll have to figure out soon is if we can update the app color dynamically as the user selects custom theme colors. Otherwise we'll want a neutral color for the themeColor.

Reply:
Ah good catch. I had just taken the color from one of the cards rather than the top bar.